style.css

```diff
@@ -0,0 +1,334 @@
+@media (max-width: 319px) {
+
+}
+
+@media (min-width: 320px) and (max-width: 374px) {
+
+}
+
+@media (min-width: 375px) and (max-width: 767px) {
+
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+	.wrapper {
+		width: 768px;
+	}
+	.header {
+		grid-template-columns: 1fr 1fr;
+	}
+	.header__menu, .header__search, .header__user-block, .header__cart, .header__language-block {
+		display: none;
+	}
+	.menu-bar {
+		display: flex;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)